src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/AttributeSnippetFormatOnAutoInsertProvider.cs

```diff
@@ -20,7 +24,9 @@ internal class AttributeSnippetOnAutoInsertProvider : RazorOnAutoInsertProvider
 
         public override string TriggerCharacter => "=";
 
-        public AttributeSnippetOnAutoInsertProvider(TagHelperFactsService tagHelperFactsService)
+        public AttributeSnippetOnAutoInsertProvider(
+            TagHelperFactsService tagHelperFactsService, ILoggerFactory loggerFactory) :
+            base(loggerFactory)
```

Comment:
Nitty nitty nit nit (I guess I'm in a mood?), but this looks very strange to me.. You're not wrapping the arguments, but they're on the their own line?  I would do this:

```suggestion
        public AttributeSnippetOnAutoInsertProvider(TagHelperFactsService tagHelperFactsService, ILoggerFactory loggerFactory)
            : base(loggerFactory)
```

or 

```suggestion
        public AttributeSnippetOnAutoInsertProvider(
            TagHelperFactsService tagHelperFactsService, 
            ILoggerFactory loggerFactory)
            : base(loggerFactory)
```

Reply:
Oddly, I've just seen both of these two forms in this PR, and they look much better.. in particular the colon placement. Maybe this should be promoted from nit to "consistency issue"?